### DISCUSSION TITLE
M2 solid blue, unable to get into bootloader mode

### DISCUSSION THREAD

**Discussion by jn887** [bitcraze/discussions#998] [Q&A] [answered]
Hi, my Crazyflie 2.1 when powered on just has the M2 light stuck on solid blue, no other lights.  I've tried holding down the power button with no power connected, then plugging in USB power and waiting, but there is no change in the lights: M2 stays lit, nothing blinks.

Measured VCC with USB powering it and it reads 0.03VDC.  

Thanks for any guidance as to what I can do to try to fix it.

**Comment by tobbeanton** [comment]
Is something geting hot?

**Comment by jn887** [comment]
Hi Tobias,

No, nothing is getting hot.  If I leave the USB power on, M2 just stays blue but nothing is getting hot.

Thank you.

**Comment by knmcguire** [comment]
Did you do anything before this happened? Did you use a DFU flash through usb to flash the crazyflie? Or did you only use flashing over air (through crazyradio). At least m2 should be flashing when you tried to put it in bootloader mode I recon.. but just double checking as this is a common mistake.

**Comment by jn887** [comment]
This Crazyflie was flashed over the air using the crazyradio and it was able to fly a few times.  There weren't any strange messages after the firmware flashing.  It just happened that one time we tried to power it on and then noticed the power button doesn't seem to do anything and M2 just turns on solid blue as soon as power is applied.  The power button doesn't seem to do anything and when I press and hold the power button down, then apply power while still keeping the power button down, nothing changes, M2 stays solid blue.

**Reply by tobbeanton** [reply, marked as answer]
Can you power on/off with the power button?

It smells hardware issue. Can you send an email to support@bitcraze.io and we will arrange a replacement.

**Reply by jn887** [reply]
No, the power button doesn't do anything.  The blue M2 light stays on and solid, doesn't even blink when the power button is pressed.

Thank you so much for your help, I'll email support.

**Reply by jn887** [reply]
Hi Tobias, I've received the replacement and it works beautifully!  Thank you so much for your wonderful help.  Our CF flock are now all working very well and I've shipped back the faulty one. Thank you!
